### DISCUSSION TITLE
guide/features/encryption/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#195] [docs-comment]
# guide/features/encryption/

加密 在本主题中，支持 全站加密 和 部分加密 等多种灵活的加密方式。 提示 由于 vuepress 是静态站点，其自身限制的原因，加密 仅仅只是 看起来 看不到内容， 并且在 编译时，不再将 内容 预渲染到 html 中，但实际上 还是能够从 站点源文件 中获取到内容。 因此，不建议将 加密 功能 认为是 安全可靠的。 请尽量避免将 加密功能 应用于...

https://theme-plume.vuejs.press/guide/features/encryption/

**Comment by xiaowiba** [comment]
解密后多久会重新加密呢，如何解除已解密状态呢？

**Reply by pengzhanbo** [reply]
在网站会话期间保持解密，会话结束就过期。会话就是指你打开网站页面到你关闭所有的相关的标签页面。

**Reply by xiaowiba** [reply]
感谢解答，希望未来可增加更多配置，比如过期时间设置等等

**Reply by xiaowiba** [reply]
打错字了，是 过期时间设置 等等

**Comment by hub-yu** [comment]
部分内容加密功能输入正确密码后，内容不能正常显示

**Reply by pengzhanbo** [reply]
该问题在新版本中已修复
